src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/DefaultHtmlGenerator.cs

```diff
@@ -802,7 +802,7 @@ public virtual IHtmlContent GenerateAntiforgery(ViewContext viewContext)
             {
                 foreach (var modelError in modelState.Errors)
                 {
-                    var errorText = ValidationHelpers.GetUserErrorMessageOrDefault(modelError, modelState: null);
+                    var errorText = ValidationHelpers.GetUserErrorMessageOrDefault(modelError, modelState);
```

Comment:
Checked MVC 5 behavior. It does not ignore the errors in validation summary.

![image](https://cloud.githubusercontent.com/assets/1579269/10153086/95be95c0-660d-11e5-8128-477a9a3bf3e1.png)


Looked at the code and there is a special casing in `DefaultModelBinder` for Format and Overflow exceptions which adds the invalid messages. http://aspnetwebstack.codeplex.com/SourceControl/latest#src/System.Web.Mvc/DefaultModelBinder.cs
```C#
// We only consider "known" type of exception and do not make too aggressive changes here
if (exception is FormatException || exception is OverflowException)
{
    string displayName = propertyMetadata.GetDisplayName();
    string errorMessageTemplate = GetValueInvalidResource(controllerContext);
    string errorMessage = String.Format(CultureInfo.CurrentCulture, errorMessageTemplate, modelState.Value.AttemptedValue, displayName);
    modelState.Errors.Remove(error);
    modelState.Errors.Add(errorMessage);
    break;
}
```